### ISSUE TITLE
Avoided redundant navigation in afterLogin

### ISSUE THREAD

**Issue by Glandos** [d0whc3r/vue-auth-plugin#282]
**Describe the bug**
Similar to #265, there can be a redundant navigation in the `afterLogin` method. 

**To Reproduce**
1. Add a form onto the `/` path
2. Log in
3. See the error:
```
 Uncaught (in promise) Error: Avoided redundant navigation to current location: "/".
    createRouterError http://localhost:8080/js/chunk-vendors.js:76180
    createNavigationDuplicatedError http://localhost:8080/js/chunk-vendors.js:76153
    confirmTransition http://localhost:8080/js/chunk-vendors.js:76313
    transitionTo http://localhost:8080/js/chunk-vendors.js:76243
    push http://localhost:8080/js/chunk-vendors.js:76610
    push http://localhost:8080/js/chunk-vendors.js:77031
    push http://localhost:8080/js/chunk-vendors.js:77030
```

**Expected behavior**
No error

**Additional context**
This could use the new `push` method introduced in d538f17562acaa1f9814697afd07046a22fd2c37. I didn't want to propose a PR, because I don't really know if I can just replace `this.router.push` by `this.push` in `afterLogin`.

**Comment by d0whc3r** [comment]
Ok, I will check asap

**Comment by d0whc3r** [comment]
It should be fixed in version 1.8.5
